### ISSUE TITLE
Overly complex documentation and legal considerations

### ISSUE THREAD

**Issue by tophermurphy** [Myndex/SAPC-APCA#141]
> BridgePCA was developed to address some aspersions from a small group of trolls

I'm just a developer, trying to improve accessibility on the websites I build. The full documents are too complex for me to put into use.

I'll wait (who knows how long) for WCAG 3 guidelines are finalized before using the APCA color contrast guidelines.

In the meantime, try not to insult your audience.

**Comment by tophermurphy** [comment]
> the truth is that WCAG 2 is not codified as an absolute law in most places (check your region). The reality is actual accessibility is what is legal

I would consult a lawyer about this line. WCAG is industry standard and used in legal cases. There may be precedent here. I don't know either way, I'm not a lawyer.

**Comment by Myndex** [comment]
@tophermurphy said

> *In the meantime, try not to insult your audience*.

Hi @tophermurphy,

I am sorry—not directed at you by any means. 

I'm not intending to insult my audience, and I'm concerned that it came across that way. My apologies.

### Background
What is going on is a few toxic individuals have been attacking me personally, this seems to be related to our research group demonstrating that WCAG 2 contrast is intellectually dishonest, not based on science nor research. The "trolls" continue to make false claims regarding APCA and actual accessibility. But in actual accessibility, WCAG 2 fails and it's easy to demonstrate. APCA was developed to address this problem.

The "trolls" I mention are the couple people that have targeted me personally with the intent to obstruct our work. Certainly not referring to my general audience and I'm sorry that it came across that way.

BridgePCA was developed specifically to address their claims that "by the numbers" APCA does not match WCAG 2 in some edge cases where WCAG 2 is significantly wrong. Technically speaking, their claims are not true—APCA is overall stricter than WCAG 2 and supports actual accessibility.

**Anyway, it didn't work:** they apparently have it out for me personally, and they keep moving goal posts. I've given up trying to fix the WCAG 2-3 problem, and brought the project elsewhere.

Nevertheless, thank you for bringing this to my attention. I don't know how to deal with the derision and obstruction from those people, and apparently I am easily triggered.

As a side note, I am autistic, and sometimes my communication is not received as intended. This is something I struggle with, and again I apologize how it came across.

### Of Law and Opinion
@tophermurphy said
> *I would consult a lawyer about this line. WCAG is industry standard and used in legal cases. There may be precedent here. I don't know either way, I'm not a lawyer*

No relevant legal precedent. WCAG2 was copy/pasted to parts of the EU 301, and the USAccess's 508 government guidelines uses it, but with some important exclusions, including "alternative means".

I have studied law, I was planing to enroll in law school when I lost my eyesight circa 2013. 

The bright line is actual accessibility. And I have  discussed this with accessibility attorneys. This is particularly true of the ADA. I can't speak to the EU's current stance as that is somewhat fluid.

WCAG 2 was never intended to be codified as law—it has neither the vetting nor the supporting science and research behind it to support it as statute law. It was written as a voluntary "guideline" with no legal force intended. The original lead author has stated so specifically.

Lawsuits that are brought are about actual accessibility. There is no citable precedent, unless it was recent (have not checked). WCAG 2 has yet to prevail in a court of law on merits on the issue of contrast, only some out of court settlements that I have read, and they were about actual accessibility not arbitrary values IIRC. The one weak ruling in Florida years ago was reversed and moot, and is not citable case law by any means. 


### BridgePCA
WCAG 2 uses brute force to increase contrast on lighter colors beyond what is needed, and yet still grossly under-performs on dark color pairs by up to 250%. BridgePCA fixes the dark colors, but maintains the brute forcing on the light colors, so that you'll never go under 4.5:1 or 3:1, depending on which you need.

The problem here, is it presents creating reasonable contrast levels to help organize content. Not everything should be be at "high contrast", there needs to be contrast modulation as a part of design, so the eye is directed to the important content.

If you are interested, here's a key example relating to color vision, and how WCAG2 reports worse color pairs while rejecting better color pairs. Free link bypasses paywall: [Whats Red and Black and Also Not Read](https://tangledweb.xyz/whats-red-black-also-not-read-573b9c0a97ed?source=friends_link&sk=4e45dacc411aeef695190e0fb17a5bc7)

Thank you for reading,

Andy
